sw/host/opentitanlib/src/util/openocd.rs

```diff
@@ -137,8 +134,25 @@ impl OpenOcdServer {
             return Ok(());
         };
 
-        // TODO: send "shutdown" command and wait before killing the process.
-        Ok(child.kill()?)
+        // Ask the server nicely to shut down.
+        self.send_tcl_cmd("shutdown")
+            .context("failed to send shutdown command")?;
+
+        // Give it some time to shut down, otherwise kill it.
+        let start = Instant::now();
+        let timeout = Duration::from_secs(2);
```

Comment:
same here, named const

Reply:
Removed this delay in favour of using `.wait()` for `fn stop` and an immediate `.kill()` for `fn drop`.